airbyte-integrations/connectors/source-sendgrid/sample_files/no_spam_reports_configured_catalog.json

```diff
@@ -411,32 +382,6 @@
       "sync_mode": "incremental",
       "cursor_field": ["created"],
       "destination_sync_mode": "append"
-    },
```

Comment:
why these two were removed?

Reply:
they are returning zero records in tests and break it.

Reply:
I see, then I think the name of the file slightly off